alert.html

```diff
@@ -0,0 +1,115 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alert</title>
+    <link rel="stylesheet" href="lift.css">
+</head>
+<body>
+    <nav class="navigation">
+        <img class="avatar size-m" src="/images/logo.png">
+         <div class="nav-brand">Lift UI</div>
+        
+         <ul class="list-non-bullet nav-pills">
+             <li class="list-item-inline">
+                 <a class="link link-active" href="https://github.com/pranjalmahajan96/component-library/tree/development">Github</a>
+             </li>           
+         </ul>
+     </nav>
+
+   <div class="wrapper">
+        <aside>
+            <li class="list-non-bullet">
+                <ul>
+                    <a class="link" href="/index.html">
+                        Getting Started
+                    </a> 
+                    </ul>
+                <ul>
+                <a class="link" href="/avatar.html">
+                    Avatar
+                </a> 
+                </ul>
+                <ul>
+                <a class="link" href="/alert.html">
+                    Alert
+                </a> 
+                </ul>
+                <ul>
+                    <a class="link" href="/badge.html">
+                        Badge
+                    </a>
+                </ul>
+                <ul>
+                    <a class="link" href="/button.html">
+                        Button
+                    </a>
+                </ul>
+                <ul>
+                    <a class="link" href="/card.html">
+                        Card
+                    </a>
+                </ul>
+                <ul>
+                    <a class="link" href="/image.html">
+                        Image
+                    </a>
+                </ul>
+                <ul>
+                    <a class="link" href="/input.html">
+                        Input
+                    </a>
+                </ul>
+                <ul>
+                    <a class="link" href="/lists.html">
+                        Lists
+                    </a>
+                </ul>
+                <ul>
+                    <a class="link" href="/navigation.html">
+                        Navigation
+                    </a>
+                </ul>
+                <ul>
+                    <a class="link" href="/modal.html">
+                        Modal
+                    </a>
+                </ul>
+                <ul>
+                    <a class="link" href="/rating.html">
+                        Rating
+                    </a>
+                </ul>
+                <ul>
+                    <a class="link" href="/slider.html">
+                        Slider
+                    </a>
+                </ul>
+                <ul>
+                    <a class="link" href="/grid.html">
+                        Grid
+                    </a>
+                </ul>
+                <ul>
```

Comment:
Hi, 

1) You may provide the active link which is clicked or may provide the hover effect when a particular link is clicked.

2) Navigation Bar could be consistent.

3) When clicked on Navigation, Utilities in Text Utilities goes to the another line. You make increase the width.

Thank you.